Tabloid/Controllers/CategoryController.cs

```diff
@@ -62,7 +62,7 @@ public IActionResult Update(int id, Category category)
                 return BadRequest();
             }
             _categoryRepository.Update(category);
-            return NoContent();
+            return Ok(category);
```

Comment:
It's common for PUT requests to return 204 No Content, or 200 Ok. I personally prefer Ok, with the content that was just modified so this is a great choice from my perspective. 

Here's more on that - https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT#responses